### PR TITLE
chore: add vrts for notification components

### PR DESCRIPTION
**Pull request by emmenko**
This will make it easier to convert css -> emotion.

image

### CODE REVIEW

**Comment by jonnybel** [comment]
It failed the tests but I just ran locally and it was working 🤔

**Comment by emmenko** [comment]
It's fixed already. We only need to approve the VRTs

**Comment by jonnybel** [comment]
Should we do it, or design? They look fine to me.

File: visual-testing-app/webpack.config.js [hunk 1]
```diff
@@ -23,9 +23,15 @@ module.exports = {
...
-        'application-components': {
-          test: /application-components/,
-          name: 'application-components',
+        '@local-build/application-components': {
+          test: /@local-build\/application-components/,
+          name: '@local-build/application-components',
+          chunks: 'all',
+          priority: -15,
+        },
+        '@local-build/application-components': {
```
Comment (montezume) [inline visual-testing-app/webpack.config.js]: why is this duplicated? 😕

Comment (emmenko) [reply visual-testing-app/webpack.config.js]: Ups, I’ll fix it later

**Pull request merged**